
From: Christoph Hellwig <hch@lst.de>

It's already marked BROKEN_ON_SMP, but even a UP compile yields tons of
errors.  While those aren't deeply complicated to fix having them for over
a year now is a pretty good indicator no one cares.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/char/Kconfig |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/char/Kconfig~mark-pcxx-as-broken drivers/char/Kconfig
--- 25/drivers/char/Kconfig~mark-pcxx-as-broken	2004-08-30 00:04:53.457221448 -0700
+++ 25-akpm/drivers/char/Kconfig	2004-08-30 00:04:53.461220840 -0700
@@ -138,7 +138,7 @@ config CYZ_INTR
 
 config DIGIEPCA
 	tristate "Digiboard Intelligent Async Support"
-	depends on SERIAL_NONSTANDARD && BROKEN_ON_SMP
+	depends on SERIAL_NONSTANDARD && BROKEN
 	---help---
 	  This is a driver for Digi International's Xx, Xeve, and Xem series
 	  of cards which provide multiple serial ports. You would need
_
